packages/manager/modules/iam/src/components/identities/groupIdentities/groupIdentities.controller.js

```diff
@@ -17,19 +20,55 @@ export default class groupIdentitiesController {
     );
   }
 
-  async getIdentityGroup(identity) {
-    try {
-      const group = await this.IAMService.getGroup(identity.id);
+  getIdentityGroup(identity) {
+    return this.IAMService.getGroup(identity.id)
+      .then((group) => group)
+      .catch((error) => {
+        const { message } = error.data ?? {};
+        return { ...identity, message };
+      });
```

Comment:
.then() callback is useless here as you don't transform the returned object

```suggestion
    return this.IAMService.getGroup(identity.id)
      .catch((error) => {
        const { message } = error.data ?? {};
        return { ...identity, message };
      });
```